src/styles/variables/_layout.scss

```diff
@@ -0,0 +1,80 @@
+//Layout variables
+$a4-width: 1000px;
+$a4-height: 1500px;
+$padding: 30px 25px;
+
+//Layout
+.cv-wrapper {
+  display: flex;
+  margin: 15px;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  position: relative;
+  width: 100%;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100%;
+  position: relative;
+  flex: 1;
+  &--orange {
+    background-color: $orange;
+    position: relative;
+    padding: $padding;
+    h1 {
+      color: $white;
+    }
+  }
+}
+
+.double-column {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100%;
+  flex: 2;
+  padding: $padding;
+}
+
+.a4-column {
+  width: $a4-width;
+  //height: $a4-height;
+  box-shadow: 0 10px 20px #888;
+}
+
+.functie {
```

Comment:
Je gebruikt hier nederlandse classnaam, maar in het overgrote deel van je document gebruik je engels termen. Probeer consistent te zijn in je naam geving.